src/scatter_buildblock/ScatterEstimation.cxx

```diff
@@ -439,15 +497,7 @@ set_up()
     if (input_projdata_sptr->get_num_segments() > 1)
     {
         info("ScatterEstimation: Running SSRB on input data...");
-        shared_ptr<ProjDataInfo> proj_data_info_2d_sptr(
-                    SSRB(*this->input_projdata_sptr->get_proj_data_info_sptr(),
-                          this->input_projdata_sptr->get_num_segments(), 1, false));
-
-	this->input_projdata_2d_sptr.reset(new ProjDataInMemory(this->input_projdata_sptr->get_exam_info_sptr(),
-                                                                 proj_data_info_2d_sptr));
-
-        SSRB(*this->input_projdata_2d_sptr,
-             *input_projdata_sptr,false);
+        this->input_projdata_2d_sptr.reset(new ProjDataInMemory(*make_2D_projdata(this->input_projdata_sptr)));
```

Comment:
now we're making a copy of a copy... instead
```suggestion
        this->input_projdata_2d_sptr = make_2D_projdata(this->input_projdata_sptr));
```

Reply:
why was this resolved? The code is unchanged. (sorry, I probably lost track). There's a few more like this below.

Reply:
looks like you did. GitHub tripped me up by going to an old commit. sorry.

Reply:
Ah OK